### ISSUE TITLE
Set updated_at when executing counter_culture_fix_counts

### ISSUE THREAD

**Issue by AlexVPopov** [magnusvk/counter_culture#210]
```ruby
class Foo < AR::Base
  has_many :bars
end

class Bar < AR::Base
  belongs_to :foo
  counter_culture :foo, touch: true
end
```

and

```ruby
Bar.counter_culture_fix_counts
```

I think in this case also the `updated_at` field of the `foos` needs to be updated. 

Does this make sense to you?

**Comment by AlexVPopov** [comment]
Currently, as I workaround, in a migration, I'd put `Foo.find_each(&:touch)`.

**Comment by magnusvk** [comment]
Yeah, that makes sense. Feel free to submit a PR. 😄

**Comment by AlexVPopov** [comment]
@magnusvk I'm willing to submit a PR. What I've done so far:
* forked the project;
* cloned locally the fork;
* ran `rspec spec`

I get the following failure:

```bash
Failures:

  1) CounterCulture should fix a STI counter cache correctly
     Failure/Error: expect(company.twitter_reviews_count).to eq(1)

       expected: 1
            got: 0

       (compared using ==)
     # ./spec/counter_culture_spec.rb:1018:in `block (2 levels) in <top (required)>'

Finished in 2.31 seconds (files took 1.55 seconds to load)
35 examples, 1 failure

Failed examples:

rspec ./spec/counter_culture_spec.rb:1003 # CounterCulture should fix a STI counter cache correctly
```

After some investigation I found out that the failure is cause by [the `after_create` callback in `Review`](https://github.com/magnusvk/counter_culture/blob/master/spec/models/review.rb#L20). When I comment out this callback, the test passes. Also, if I put the callback after:

```ruby

# TwitterReview
counter_culture [:user, :manages_company]
```

that is, I move it from `Review` to `TwitterReview` (for the sake of executing it after counter_culter has finished updating the counters) the test also passes. I am currently investigating this, but it appears that after `after_create :update_some_text` is ran, `@_counter_culture_active` is not properly set to `false` in `_wrap_in_counter_culture_active(&block)`, so that when it gets to decide whether to update the counter for `twitter_reviews_count` in Company, `@_counter_culture_active` is still `true` and thus does not execute the block.

Before I continue my investigation - can you tell me if the test fails on your side as well?

I'm running ruby 2.4.1 and rails 5.1.5.

https://github.com/magnusvk/counter_culture/blob/master/lib/counter_culture/extensions.rb#L78

**Comment by AlexVPopov** [comment]
O.K., this does not happen on 5.1.4., it only happens on 5.1.5. I think the problem is somehow related to https://github.com/rails/rails/pull/27780

**Comment by magnusvk** [comment]
Yeah, this used to work fine with Rails 5.1. I haven’t had a chance to look into this yet but will try to soon. Thanks for digging. For purposes of your PR feel free to just test against 5.1.4 for now where the test is green.

**Comment by magnusvk** [comment]
Fixed by #212
